### PR TITLE
refactor: refactor the default light theme

### LINKED ISSUE #26: [Feature] common theme api by series mark name
### What problem does this feature solve?

common theme properties in ITheme 

### What does the proposed API look like?

```
export interface ITheme {
  /** 全局 mark 样式属性，按 mark 名称区分 */
  markByName?: IGlobalMarkThemeByName;
}

export interface IGlobalMarkThemeByName {
  /** used in lineSeries, areaSeries, radarSeries, etc. */
  line?: Partial<IMarkTheme<ILineMarkSpec>>;
  /** used in lineSeries, areaSeries, radarSeries, scatterSeries etc. */
  point?: Partial<IMarkTheme<ISymbolMarkSpec>>;
  /** used in lineSeries, areaSeries, radarSeries, etc. */
  area?: Partial<IMarkTheme<IAreaMarkSpec>>;
  /** used in barSeries, rangeColumnSeries etc. */
  bar?: Partial<IMarkTheme<IRectMarkSpec>>;
  /** used in many series */
  label?: Partial<IMarkTheme<ITextMarkSpec>>;

  [markName: string]: Partial<IMarkTheme<any>>;
}

```

### PR DESCRIPTION
**Pull request by kkxxkk2019**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #26 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -90,6 +90,13 @@
       "description": "Run this command to start vchart development server",
       "shellCommand": "rush run -p @visactor/vchart -s start"
     },
+    {
+      "commandKind": "global",
```
Comment (kkxxkk2019) [inline common/config/rush/command-line.json:94]: 删掉吧

File: packages/vchart/package.json [hunk 2]
```diff
@@ -23,7 +23,8 @@
     "test-cov": "jest -w 16 --coverage",
     "test-live": "npm run test-watch __tests__/unit/",
     "test-watch": "DEBUG_MODE=1 jest --watch",
-    "ci": "node github-ci.js"
+    "ci": "node github-ci.js",
+    "demo": "vite serve examples"
```
Comment (kkxxkk2019) [inline packages/vchart/package.json:27]: 删掉~

File: packages/vchart/package.json [hunk 3]
```diff
@@ -86,7 +87,9 @@
     "d3-hierarchy": "^3.1.1",
     "vite": "3.2.6",
     "node-fetch": "2.6.7",
-    "form-data": "~4.0.0"
+    "form-data": "~4.0.0",
+    "vite-plugin-markdown": "^2.1.0",
```
Comment (kkxxkk2019) [inline packages/vchart/package.json:91]: 删掉

File: packages/vchart/package.json [hunk 4]
```diff
@@ -86,7 +87,9 @@
...
+    "markdown-it": "13.0.1"
```
Comment (kkxxkk2019) [inline packages/vchart/package.json:92]: 删掉

File: packages/vchart/src/theme/buildin-theme/light/color-scheme.ts [hunk 5]
```diff
@@ -48,7 +48,8 @@ export const colorScheme: IThemeColorScheme = {
     palette: {
       labelFontColor: '#89909D',
       titleFontColor: '#000000',
-      axisGridColor: '#F1F2F5'
+      axisGridColor: '#EBEDF2',
```
Comment (kkxxkk2019) [inline packages/vchart/src/theme/buildin-theme/light/color-scheme.ts:51]: 这个命名感觉不太通用，主要线颜色、次要线颜色、辅助线颜色这种会不会更好一些？

Comment (kkxxkk2019) [reply packages/vchart/src/theme/buildin-theme/light/color-scheme.ts:51]: theme 的相关文档里记得把 markName 这个加上

Comment (kkxxkk2019) [reply packages/vchart/src/theme/buildin-theme/light/color-scheme.ts:51]: > 这个命名感觉不太通用，主要线颜色、次要线颜色、辅助线颜色这种会不会更好一些？

或者类似于这种：

image

Comment (zamhown) [reply packages/vchart/src/theme/buildin-theme/light/color-scheme.ts:51]: 好的，目前设计那边还没有详细区分，我先记个issue吧

File: packages/vchart/tsconfig.json [hunk 6]
```diff
@@ -17,5 +17,5 @@
       "module": "commonjs"
     }
   },
-  "include": ["src"]
```
Comment (kkxxkk2019) [inline packages/vchart/tsconfig.json:20]: 为啥要加上 __tests__ ?

**Review by zamhown — APPROVED** [review]

**Pull request merged**